index.md

```diff
@@ -1,213 +1,270 @@
+
+                <h4>Test</h4>
+
+                <ul>
+                    <li>Unit testing</li>
+                    <li>Integration testing</li>
+                    <li>On-device testing</li>
+                </ul>
+
+                <h4>Debug</h4>
+
+                <ul>
+                    <li>IDE debugger</li>
+                    <li>Web-based debugger</li>
+                    <li>async/await aware</li>
+                    <li>Expression evaluator</li>
+                </ul>
+
+                <h4>Profile</h4>
+
+                <ul>
+                    <li>Timeline</li>
+                    <li>CPU and memory</li>
+                    <li>In-app perf charts</li>
+                </ul>
+            </div>
+
+            <div class="feature-list-group">
+                <h3>Deploy</h3>
+
+                <h4>Compile</h4>
+
+                <ul>
+                    <li>Native ARM code</li>
+                    <li>Dead code elimination</li>
+                </ul>
+
+                <h4>Distribution</h4>
 
-## What is Flutter?
+                <ul>
+                    <li>App Store</li>
+                    <li>Play Store</li>
+                </ul>
+            </div>
 
-Flutter is a mobile app SDK for building high-performance, high-fidelity,
-apps for iOS and Android, from a single codebase.
+        </div>
 
-The goal is to enable developers to deliver high-performance apps that feel natural
-on different platforms. We embrace differences in scrolling behaviors,
-typography, icons, and more.
+        <p>
+        Learn more about what makes Flutter special in the
+        <a href="/technical-overview/">technical overview</a>.
+        </p>
+    </section>
 
-<object type="image/svg+xml" data="images/whatisflutter/hero-shrine.svg" style="width: 100%; height: 100%;"></object>
+    <section class="homepage__try_flutter">
 
-This is a demo app from the [Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery/lib/demo),
-a collection of Flutter sample apps you can run after installing Flutter and setting
-up your environment. Shrine has high-quality scrolling images, interactive cards,
-buttons, dropdown lists, and a shopping cart page. To look at the single codebase
-for this and more examples, [visit our GitHub repository](https://github.com/flutter/flutter/tree/master/examples).
-
-No mobile development experience is required to get started. Apps are written
-in [Dart](https://dartlang.org/), which will look familiar if you've used a
-language like Java or JavaScript. Experience with object-oriented languages
-is definitely helpful, but even non-programmers have made Flutter apps!
+        <div class="homepage__try_today">Try Flutter today. Getting started is easy.</div>
 
-## Why use Flutter?
+        <div class="homepage__button_row">
+        <a href="/getting-started/" class="get-started-button">Get Started</a>
+        {% include note.html content="_Flutter is an alpha, open-source project_." %}
```

Comment:
We used to have that, but now there's a Support link on every page on the website.